$MirOS: ports/devel/autoconf/2.61/patches/patch-bin_autoheader_in,v 1.1 2007/05/08 20:38:25 tg Exp $
--- bin/autoheader.in.orig	2008-08-28 04:08:10.000000000 +0200
+++ bin/autoheader.in	2009-07-14 21:24:23.000000000 +0200
@@ -205,7 +205,7 @@ my $out = new Autom4te::XFile ("> " . op
 
 # Don't write "do not edit" -- it will get copied into the
 # config.h, which it's ok to edit.
-print $out "/* $config_h_in.  Generated from $ARGV[0] by autoheader.  */\n";
+print $out "/* $config_h_in.  Generated from $ARGV[0] by autoheader @VERSION@.  */\n";
 
 # Dump the top.
 if ($config_h_top)
